### PR TITLE
Astro CLI 1.12 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

**Review by neel-astro — APPROVED** [review]
Verified changes related to `airflow.expose_port`, LGTM. Thanks ❤️

**Review by sunkickr — APPROVED** [review]

File: astro/cli/release-notes.md [hunk 1]
```diff
@@ -17,6 +17,33 @@ id: release-notes
...
+
+### Additional improvements
+
+- You can now expose your local Airflow webserver and postgres database to all networks you're connected to using the following command:
+
+    ```sh
+    astro config set airflow.expose_port true
+    ```
+
+- When you trigger a DAG deploy to Astro, the CLI now includes the name of the DAG bundle version that it pushed. You can use this name to verify that your Deployment is running the correct version of your DAGs after a deploy.
```
Comment (lzdanski) [inline astro/cli/release-notes.md:32]:
```suggestion
- When you trigger a DAG deploy to Astro, the CLI now includes the name of the DAG bundle version that it pushed. You can use this name to verify that your Deployment uses the correct version of your DAGs after a deploy.
```

File: astro/cli/release-notes.md [hunk 2]
```diff
@@ -17,6 +17,33 @@ id: release-notes
...
+    ```sh
+    astro config set disable_astro_run true
+    ```
+  
+- In new Astro projects, `requirements.txt` now includes a commented list of the pre-installed provider packages on Astro Runtime. 
+
+### Bug fixes
+
+- Fixed an issue where the default DAG integrity test would sometimes generate an error for valid uses of `os.getenv(key,default)`.
+- Fixed a few bugs in the default Astro project DAGs.
```
Comment (lzdanski) [inline astro/cli/release-notes.md:45]:
```suggestion
- Fixed bugs in the default Astro project DAGs.
```

**Review by lzdanski — APPROVED** [review]
Editorial pass - I only have nitpicky active/passive voice feedback

**Pull request merged**